### PR TITLE
Add Alex as a NPM dependency

### PR DESCRIPTION
**Pull request by bencgreenberg**
## Description

Related to DEVX-2258 and to https://github.com/Nexmo/nexmo-developer/pull/3405 this adds [alex](https://github.com/get-alex/alex) as a dependency in Station, similar to the spellcheck package.

## Deploy Notes

Notes regarding deployment the contained body of work. These should note any db migrations, etc.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,8 +14,9 @@
     "@babel/preset-env": "^7.12.7",
     "@rails/ujs": "^6.0.3-4",
     "@rails/webpacker": "^5.2.1",
-    "algoliasearch": "^4.8.2",
     "@vonagevolta/volta2": "^0.1.5",
+    "alex": "^9.1.0",
```
Comment (mheap) [inline package.json:18]: Alex is a dev dependency, not a runtime dependency

Comment (bencgreenberg) [reply package.json:18]: Good catch! I'll fix.

Comment (bencgreenberg) [reply package.json]: Done :)

**Review by fabianrbz — APPROVED** [review]

**Review by fabianrbz — CHANGES_REQUESTED** [review]
@bencgreenberg looks like the deploy failed due to an outdated `yarn.lock`, can you check that?

**Comment by bencgreenberg** [comment]
@fabianrbz fixed the yarn issue

**Review by fabianrbz — APPROVED** [review]

**Pull request merged**